go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ethersphere/bee
 
-go 1.16
+go 1.14
```

Comment:
I know it's a revert but I wonder if we should stay on 1.15 since our workflows use this ver?

Reply:
yeah it wouldn't hurt